.gitignore

```diff
@@ -404,3 +404,4 @@ FodyWeavers.xsd
 # Custom
 **/alarmLog.txt
 **/scadaConfig.txt
+**/alarmConfig.txt
```

Comment:
We don't need this, `ScadaConfig` line 7 should have the `DefaultAlarmLogPath` be `"./Data/alarmLog.txt"` instead of `"./Data/alarmConfig.txt"`, I forgot to commit the fix.

Reply:
https://github.com/magley/untitled-scada-project/pull/31/commits/058139f65b4c555be53b238b216b91ed90dbfac5